README.md

```diff
@@ -82,6 +86,10 @@ sudo ig run trace_open:latest
 
 Check [Installing on Linux](https://www.inspektor-gadget.io/docs/latest/reference/install-linux) to learn more.
 
+`ig` is also packaged for the following distributions:
+
+[![`ig`](https://repology.org/badge/vertical-allrepos/inspektor-gadget.svg)](https://repology.org/project/inspektor-gadget/versions)
```

Comment:
The documentation should be updated to have a section about installing distro packages:
https://inspektor-gadget.io/docs/latest/reference/install-kubernetes#installing-kubectl-gadget
https://inspektor-gadget.io/docs/latest/reference/install-linux/